composer.json

```diff
@@ -46,7 +46,7 @@
 	"require": {
 		"php": ">=7.3",
 		"composer/installers": ">=1.0.1",
-		"mediawiki/semantic-media-wiki": "~3.0|~4.0|~5.0",
+		"mediawiki/semantic-media-wiki": "~3.0|~4.0|~5.0|dev-master",
```

Comment:
This is not valid

Reply:
Revert:
* https://github.com/SemanticMediaWiki/SemanticResultFormats/pull/888

Reply:
@JeroenDeDauw are you sure this is not "valid"? It's "discouraged", because dev-master is a moving target. I think this came in during the massive rework on both sides, SMW and SRF to see, if it's working with the latest changes in dev-master